Android--EN/domain/RectangleConfig.java

```diff
@@ -40,8 +40,8 @@ public RectangleConfig(Double width, Double height, AnimationMode mode) {
 
     /**
      * The `RectangleConfig` constructor.
-     * <p>
-     * In this method, pass in the `width`, `height`, and `mode` parameters. Based on the `width` and `height` you pass in, the SDK calculates `originX` and `originY`, the X and Y coordinates of the top left corner of the view rectangle in the world coordinate system, in the following ways:
+     * <p> // TODO William Why do you use paragraph tags sometimes but not others? Does it matter? Is there some variation in output?
+     * In this method, pass in the `width`, `height`, and `mode` parameters. Based on the `width` and `height` you pass in, the SDK calculates `originX` and `originY`, the X and Y coordinates of the top left corner of the view rectangle in the world coordinate system, in the following ways: // TODO William Does this method in fact pass in a 'mode' parameter? Isn't that excluded here? Or is it passing in a default value from somewhere?
```

Comment:
Nice catch! There is no 'mode' parameter in this method. The method has a default mode, which is Continuous.